frontend/packages/console-app/src/components/nodes/NodeStatus.tsx

```diff
@@ -1,16 +1,67 @@
 import * as React from 'react';
+import * as _ from 'lodash';
+import ConsumerPopover from '@console/shared/src/components/dashboard/utilization-card/TopConsumerPopover';
 import { Status, SecondaryStatus, getNodeSecondaryStatus } from '@console/shared';
 import { NodeKind } from '@console/internal/module/k8s';
+import { humanizeBinaryBytes, humanizeNumber } from '@console/internal/components/utils';
 import { nodeStatus } from '../../status/node';
+import { PressureQueries, Condition } from '../../queries';
+
+const conditionDescriptionMap = Object.freeze({
+  [Condition.DISK_PRESSURE]: 'available memory is low',
+  [Condition.MEM_PRESSURE]: 'available disk capacity is low',
+  [Condition.PID_PRESSURE]: 'CPU is running a large number of processes',
+});
+
+const humanizeMap = Object.freeze({
+  [Condition.DISK_PRESSURE]: humanizeBinaryBytes,
+  [Condition.MEM_PRESSURE]: humanizeBinaryBytes,
+  [Condition.PID_PRESSURE]: humanizeNumber,
+});
+
+const isMonitoredCondition = (condition: string): boolean =>
+  ['DiskPressure', 'PIDPressure', 'MemoryPressure'].includes(condition);
```

Comment:
Did you want to reference the same strings `Condition.DISK_PRESSURE`, `Condition.MEM_PRESSURE`, `Condition.PID_PRESSURE` in this func instead?

Reply:
Yeah, that would be better. Thanks.